binlogctl/nodes_test.go

```diff
@@ -93,6 +100,13 @@ func (s *nodesSuite) TestUpdateNodeState(c *C) {
 	c.Assert(findNode, IsTrue)
 }
 
+func (s *testNodesSuite) TestCreateRegistry(c *C) {
+	urls := "127.0.0.1:2379"
+	registry, err := createRegistry(urls)
+	c.Assert(err, IsNil)
+	c.Assert(registry, NotNil)
```

Comment:
What else can we check about the returned `registry`?

Reply:
add in https://github.com/pingcap/tidb-binlog/pull/604/commits/c4445f909466bab2b0432542146c4b1258943ab2